src/Our.Umbraco.NestedContent/Properties/VersionInfo.cs

```diff
@@ -14,6 +14,6 @@
 using System.Runtime.InteropServices;
 
 [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyInformationalVersion("1.0.0-local-000100")]
+[assembly: AssemblyInformationalVersion("1.0.6-beta")]
```

Comment:
I'd like to have this reverted, it's not a big issue, more semantics.  AppVeyor CI will overwrite the version number per build anyway.